### PR TITLE
adds a contributor guideline link above the issues, as per suggestion…

### PR DESCRIPTION
**Pull request by ckhatri**
… at issue #476 

Added some css styling on font size, went with a font size of 14px.
The contributor guideline link will only show up if the .yml file has a contribute: link, so it's optional. 

I didn't use the GitHub Api for existing projects but that might be something I do later.

Hope everything checks out, let me know!

### CODE REVIEW

**Comment by shiftkey** [comment]
@ckhatri i like where this is heading. I tested it out with a project to see how it would render:



Could we put this below the label? And maybe make it visually different to the label element?

**Comment by ckhatri** [comment]
Yup! Will get that done this weekend.

**Comment by ckhatri** [comment]
uh better late than never?

File: stylesheets/stylesheet.css [hunk 1]
```diff
@@ -522,6 +522,11 @@ form {
   word-break: break-all;
 }
 
+.projects .contribute {
+  font-size: 14px;
+  word-break: break-all;
```
Comment (shiftkey) [inline stylesheets/stylesheet.css:527]: There's some weird trimming here:



Should this be `keep-all`?

File: _includes/scripts.html [hunk 2]
```diff
@@ -23,7 +23,12 @@
     </tr>
     <tr>
        <td class="details">
+
         <p class="label"><a href="<%- project.upforgrabs.link %>" title="View open issues for <%-project.name %>"><%-project.upforgrabs.name %></a></p>
+       <br>
+       <% if (project.contribute) { %>
```
Comment (shiftkey) [inline _includes/scripts.html:29]: Could you add this field to the example payload in the `README.md`?

**Review by shiftkey — CHANGES_REQUESTED** [review]
This is close - at this point I'm just thinking about whether we can style that better.

**Comment by shiftkey** [comment]
Closing this out due to inactivity.

@ckhatri if you're still interested in implementing this, please open a fresh pull request

**Pull request closed without merging**